blocks/soapy/include/gnuradio-4.0/soapy/Soapy.hpp

```diff
@@ -201,6 +213,14 @@ This block supports multiple output ports and was tested against the 'rtlsdr' an
         for (std::size_t i = 0UZ; i < nChannels; i++) {
             _device.setCenterFrequency(SOAPY_SDR_RX, rx_channels->at(i), rx_center_frequency->at(std::min(i, nFrequency - 1UZ)));
         }
+        std::vector<double> rx_center_frequency_actual;
+        for (std::size_t i = 0UZ; i < nChannels; i++) {
+            rx_center_frequency_actual.push_back(_device.getCenterFrequency(SOAPY_SDR_RX, rx_channels->at(i)));
+        }
+        auto equalWithinOnepercent = [](const std::vector<double>& a, const std::vector<double>& b) { return a.size() == b.size() && std::equal(a.begin(), a.end(), b.begin(), [](double x, double y) { return std::abs(x - y) <= 0.01 * std::max(std::abs(x), std::abs(y)); }); };
```

Comment:
lambda duplicates `detail::equalWithOnePercent()` helper function.

Reply:
in.